http/src/request/channel/reaction/mod.rs

```diff
@@ -10,3 +10,31 @@ pub use self::{
     delete_all_reactions::DeleteAllReactions, delete_reaction::DeleteReaction,
     get_reactions::GetReactions,
 };
+
+use std::fmt::Write;
+use twilight_model::channel::ReactionType;
+
+pub fn format_emoji(emoji: ReactionType) -> String {
```

Comment:
should this be `pub(crate)`?

Reply:
I think it should be

Reply:
I don't think it needs to be public, actually? It's only used in this module